webcomponents/tool/src/main/frontend/js/forms/sakai-text-input.js

```diff
@@ -0,0 +1,50 @@
+import { html } from "../assets/lit-element/lit-element.js";
+import { SakaiElement } from "../sakai-element.js";
+/**
+ * Provides a text input field, supports the maxlength attribute displaying an errorMessage when the limit is reached by the user.
+ * @example <caption>Usage:</caption>
+ * <sakai-text-input value="" id="text-input-story" maxLength="5" errorMessage="You can't insert more than 5 chars"></sakai-text-input>
+ *
+ */
+
+export class SakaiTextInput extends SakaiElement {
+  static get properties() {
```

Comment:
I know this is pedantic of me, but I hate compressing js up. I prefer to leave a blank line after class and function declarations. But, I am being pedantic - I just think it makes js look clearer. It can get messy.